core/blockchain_notries_test.go

```diff
@@ -156,9 +161,15 @@ func makeTestBackendWithRemoteValidator(blocks int) (*testBackend, *testBackend,
 
 	peer.setCallBack(func(req *requestRoot) {
 		if fastnode.validator != nil && fastnode.validator.RemoteVerifyManager() != nil {
+			resp := verifier.GetRootByDiffHash(req.blockNumber, req.blockHash, req.diffHash)
+			if failed != nil && req.blockNumber == failed.blockNumber {
+				resp.Status = failed.status
+			} else {
```

Comment:
we can remove this else condition to keep the original status that response from the verifier.